enterprise/server/remote_execution/blockio/blockio.go

```diff
@@ -163,13 +229,13 @@ type COW struct {
 
 // NewCOW creates a COW from the given chunks. The chunks should be open
 // initially, and will be closed when calling Close on the returned COW.
-func NewCOW(chunks []Store, chunkSizeBytes, totalSizeBytes int64, dataDir string) (*COW, error) {
+func NewCOW(chunks []*Chunk, chunkSizeBytes, totalSizeBytes int64, dataDir string) (*COW, error) {
```

Comment:
I don't think calling things COW is particularly clear (because Copy On Write is really a strategy / adjective describing how something else, a blockstore, works)

What if we call this NewCOWStore or similar?